### PR TITLE
parser: parse ON UPDATE CURRENT_TIMESTAMP(6)

### PR DESCRIPTION
**Pull request by tiancaiamao**
Handle ON UPDATE CURRENT_TIMESTAMP(6), which specify fraction part.

```
CREATE TABLE IF NOT EXISTS `general_log` (
  `event_time` timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6),
  `user_host` mediumtext NOT NULL,
  `thread_id` bigint(21) unsigned NOT NULL,
  `server_id` int(10) unsigned NOT NULL,
  `command_type` varchar(64) NOT NULL,
  `argument` mediumblob NOT NULL
) ENGINE=CSV DEFAULT CHARSET=utf8 COMMENT='General log'
```

### CODE REVIEW

File: parser/parser.y [hunk 1]
```diff
@@ -1448,7 +1452,7 @@ DefaultValueExpr:
 	{
 		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr("CURRENT_TIMESTAMP")}
 	}
-|	SignedLiteral
+
```
Comment (hanfei1991) [inline parser/parser.y]: remove blank

**Comment by coocood** [comment]
LGTM

**Comment by hanfei1991** [comment]
LGTM

**Comment by zimulala** [comment]
MySQL does not support the statement of `alter table t modify a timestamp ON UPDATE now;`

**Pull request merged**